src/token/api/src/types/account_identifier.rs

```diff
@@ -0,0 +1,295 @@
+use std::fmt::{Display, Formatter};
```

Comment:
This whole module is copied from IC repo. Is there a reason to do it. Why cannot we just use the IC implementation (it's already reexported by ic-helpers crate).

Reply:
That is quite interesting, yes, I am gonna check into that. I copied this file from the icrc-standard repo , so if it's exported by the IC repo, that is what am gonna use. Thanks for the tip 👍🏾